### ISSUE TITLE
the compass image animation is not fluid with 0.7.0 version

### ISSUE THREAD

**Issue by jibebec** [hemanthrajv/flutter_compass#73]
Hi when i upgraded from the 0.6.1 to 0.7.0 i noticed a different behavior : the image transitions/moves are chopped...

I can't find why. 

thank u for your help.

Regards,
JB

**Comment by GustavoNeri02** [comment]
Same here, I don't know how to fix. It's just not fluid.

**Comment by Kharda** [comment]
Yes, same here. This also affecting another plugin https://github.com/medyas/flutter_qiblah/issues/21 🙏

**Comment by iqfareez** [comment]
Same here. Is this related to the recent plugin V2 Android embedding migration?

Hope there is a fix.

**Comment by iqfareez** [comment]
I noticed that commit on master branch already fixed the issues, but the version is not yet published to pub dev. Perhaps @hemanthrajv can help publish the latest version of this packages.
